docs/features/search/how-to-guides.md

```diff
@@ -67,11 +67,22 @@ getting started guide.
 import { DefaultTechDocsCollatorFactory } from '@backstage/plugin-techdocs-backend';
 ```
 
-2. Register the `DefaultTechDocsCollatorFactory` with the IndexBuilder.
+2. If there isn't an existing schedule you'd like to run the collator on, be
+   sure to create it first. Something like...
+
+```typescript
+const every10MinutesSchedule = env.scheduler.createScheduledTaskRunner({
+  frequency: { minutes: 10 },
```

Comment:
oh, btw does this need to be `Duration.fromObject({ minutes: 10 })`

Reply:
That's one of the updates, we switched it around so that you don't need to import and add `luxon` as a dependency when configuring the schedule. That removed a bunch of steps from various parts of the backend setup.

Reply:
✨✨✨ Indeed: https://github.com/backstage/backstage/pull/11245

Reply:
I realised that I screwed up and was too quick at updating the docs! The new package update won't be out until the next/main release...
Not sure if we want to temporary revert that and keep the PR for the new "layout" open until the next release. Timing docs with main releases is going to continue to be an issue for us until we have some form of versioned docs.

Reply:
Oh nice @Rugvip!

Reply:
> I realised that I screwed up and was too quick at updating the docs! The new package update won't be out until the next/main release... Not sure if we want to temporary revert the docs updates and keep the PR for the new "layout" open until the next release. Timing docs with main releases is going to continue to be an issue for us until we have some form of versioned docs.

Hmmmm.  Yeah, good point.  These were broken even before the scheduler improvements (old version wasn't even referencing the scheduler).  ...So how about I update this to use the luxon-based version, and then I'll leave it up to y'all to handle this (and other references) as y'all deem fit?

Reply:
Sounds good 👍